### PR TITLE
Fix/dev 6209 refine homepage images

### PR DESCRIPTION
**Pull request by brett-varney**
**High level description:**

SEO audit found homepage images were not ideal; this update should improve rank and UX.

**Technical details:**

Resized large images, create WEBP versions, updated source code to use WEBP when allowed.

**JIRA Ticket:**
[DEV-6209](https://federal-spending-transparency.atlassian.net/browse/DEV-6209)

**Mockup:**
na

The following are ALL required for the PR to be merged:

Author:
- [x] Linked to this PR in JIRA ticket
- na Scheduled demo including Design/Testing/Front-end OR Provided instructions for testing in JIRA and PR `if applicable`
- [x] Verified cross-browser compatibility: Chrome, Safari, Firefox, Internet Explorer, Edge
- [x] Verified mobile/tablet/desktop/monitor responsiveness
- [x] Verified that this PR does not create any *new* accessibility issues (via Axe Chrome extension)
- na Added Unit Tests for helper functions, reducers, models and Container/Component Interactivity Expectations `if applicable` [React Testing Library](react-testing-library.md)
- na [API contract](https://github.com/fedspendingtransparency/usaspending-api/tree/dev/usaspending_api/api_contracts) updated `if applicable`
- na [Component Library Integration Status Report](https://github.com/fedspendingtransparency/data-act-documentation/blob/data-transparency-ui/frontend_apps/component-library-integration-status.md) updated `if applicable`

Reviewer(s):
- na Design review complete `if applicable`
- na [API #1234](https://github.com/fedspendingtransparency/usaspending-api/pull/1234) merged concurrently `if applicable`
- [ ] Code review complete

### CODE REVIEW

**Comment by brett-varney** [comment]
Our testing requires page to be publicly available, so it must be merged to determine the effectiveness.

File: src/js/components/homepage/features/SpendingExplorerFeature.jsx [hunk 1]
```diff
@@ -33,8 +33,8 @@ const SpendingExplorerFeature = () => (
                 <div className="feature-spending-explorer__image-wrapper feature-spending-explorer__image-wrapper_mobile">
                     <img
                         className="feature-spending-explorer__image"
+                        srcwet="img/homepage-spending-explorer.webp 1x, img/homepage-spending-explorer@2x.webp 2x"
```
Comment (MaxwellKendall) [inline src/js/components/homepage/features/SpendingExplorerFeature.jsx:36]: srcSet?

File: src/js/components/homepage/features/SpendingExplorerFeature.jsx [hunk 2]
```diff
@@ -20,8 +20,8 @@ const SpendingExplorerFeature = () => (
             <div className="feature-spending-explorer__image-wrapper feature-spending-explorer__image-wrapper_desktop">
                 <img
                     className="feature-spending-explorer__image"
+                    srcwet="img/homepage-spending-explorer.webp 1x, img/homepage-spending-explorer@2x.webp 2x"
```
Comment (MaxwellKendall) [inline src/js/components/homepage/features/SpendingExplorerFeature.jsx:23]: srcSet?

File: src/js/components/homepage/features/ProfileFeature.jsx [hunk 3]
```diff
@@ -12,23 +12,23 @@ import FeatureDropdown from './FeatureDropdown';
 
 const images = [
     {
+        srcset: 'img/homepage-profile-agency.webp 1x, img/homepage-profile-agency@2x.webp 2x',
```
Comment (MaxwellKendall) [inline src/js/components/homepage/features/ProfileFeature.jsx:15]: srcSet?

File: src/js/components/homepage/features/ImageCarousel.jsx [hunk 4]
```diff
@@ -219,8 +219,8 @@ export default class ImageCarousel extends React.Component {
                     tabIndex={-1}>
                     <img
                         className="feature-carousel-image__image"
-                        src={image.src}
                         srcSet={image.srcSet}
```
Comment (MaxwellKendall) [inline src/js/components/homepage/features/ImageCarousel.jsx:222]: is this gonna work since the datamapping is not camel case?

**Review by MaxwellKendall — CHANGES_REQUESTED** [review]
small changes!!! Spelling & casing errors

File: src/js/components/homepage/features/SpendingExplorerFeature.jsx (hunk 1 above)
Comment (brett-varney) [reply src/js/components/homepage/features/SpendingExplorerFeature.jsx:36]: That seems a sensible improvement...

**Review by MaxwellKendall — APPROVED** [review]

**Pull request merged**